Model/Method/Sofortbanking.php

```diff
@@ -52,7 +52,7 @@ public function getOrderTransactionBuilder($payment)
         $services = [
             'Name'             => 'Sofortueberweisung',
             'Action'           => 'Pay',
-            'Version'          => 1,
+            'Version'          => 0,
```

Comment:
@saranchuk-hys can you please clarify why sofort change was needed ? 
I see in LS I've used version 1

Reply:
Sorry my bad, fixed